### PR TITLE
Port submit_to helper from libyui-rake

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

File: data/targets.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:sle12:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:SLE-12"
+  obs_sr_project: "SUSE:SLE-12:Update"
+  obs_target: "SLE_12"
+:factory:
+  obs_project: "YaST:Head"
+  obs_sr_project: "openSUSE:Factory"
```
Comment (jreidinger) [inline data/targets.yml]: please add also Leap target as we already have such branch

File: lib/yast/rake.rb [hunk 2]
```diff
@@ -43,9 +26,9 @@ def self.spec_version
 
 task.enhance(prerequisites)
 
+Yast::Tasks.submit_to(:factory)
```
Comment (jreidinger) [inline lib/yast/rake.rb]: Please make it also ENV configurable, so we can use similar trick in jenkins for internal instance.

File: data/targets.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  obs_sr_project: "SUSE:SLE-12:Update"
+  obs_target: "SLE_12"
+:leap_42_1:
+  obs_api: "https://api.opensuse.org"
+  obs_target: "openSUSE_Leap_42.1"
+  obs_sr_project: "openSUSE:Leap:42.1"
+  obs_project: "YaST:openSUSE:42.1"
+:factory:
+  obs_project: "YaST:Head"
+  obs_sr_project: "openSUSE:Factory"
```
Comment (jreidinger) [inline data/targets.yml]: one more project we need. We use specific internal project to currently submit master in internal BS currently to SP1. Please check any internal machine for corect data.

**Comment by jreidinger** [comment]
LGTM

**Comment by imobach** [comment]
@jreidinger Thanks!

**Pull request merged**